ext_localconf.php

```diff
@@ -4,61 +4,49 @@
     die('Access denied.');
 }
 
+use JWeiland\Weather2\Controller\CurrentWeatherController;
+use JWeiland\Weather2\Controller\WeatherAlertController;
+use JWeiland\Weather2\Task\DeutscherWetterdienstTask;
+use JWeiland\Weather2\Task\DeutscherWetterdienstTaskAdditionalFieldProvider;
+use JWeiland\Weather2\Task\DeutscherWetterdienstWarnCellTask;
+use JWeiland\Weather2\Task\OpenWeatherMapTask;
+use JWeiland\Weather2\Task\OpenWeatherMapTaskAdditionalFieldProvider;
+use TYPO3\CMS\Extbase\Utility\ExtensionUtility;
+
 call_user_func(static function () {
-    $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['scheduler']['tasks'][JWeiland\Weather2\Task\OpenWeatherMapTask::class] = [
+    $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['scheduler']['tasks'][OpenWeatherMapTask::class] = [
```

Comment:
I would prefer to remove all these tasks and migrate them all to command API. Constructors in tasks are dangerous, you only can call a pre-configured task, but you can not set task options from CLI. Maybe let us discuss that in our projectmeeting